src/chat/components/ViewUnRead.tsx

```diff
@@ -14,6 +14,7 @@ interface ViewUnReadProps {
   customTextStyle?: StyleProp<TextStyle>;
   unReadSentMessage?: string;
   unReadSeenMessage?: string;
+  viewUnReadCustom?: () => JSX.Element;
```

Comment:
```suggestion
  viewUnReadCustom?: (hasUnread: boolean) => JSX.Element;
```
I suggest we pass in the `userUnreadMessage` prop into this custom component to allow easier customize the UI with unread/read status

Reply:
ok I updated